mlir/lib/Target/LLVMIR/DebugImporter.cpp

```diff
@@ -220,7 +230,7 @@ DINodeAttr DebugImporter::translate(llvm::DINode *node) {
   auto guard = llvm::make_scope_exit([&]() { translationStack.pop_back(); });
 
   // Convert the debug metadata if possible.
-  auto translateNode = [this](llvm::DINode *node) -> DINodeAttr {
+  auto translateNode = [this](llvm::MDNode *node) -> DINodeAttr {
```

Comment:
```suggestion
  auto translateNode = [this](llvm::DINode *node) -> DINodeAttr {
```
nit: I think this is a leftover from the MDNode change?